main.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'sinatra'
+
+# 提出時に削除する
+require 'sinatra/reloader'
+require 'debug'
+
+DB_FILE_NAME = 'db.json'
+
+before do
+  @app_title = 'メモアプリ'
+end
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+end
+
+class Memo
+  def initialize(memo_data)
+    @latest_memo_id = memo_data['latest_id']
+    @memo_lists = memo_data['memo_lists']
+    @memo_ids = @memo_lists.map(&:keys).flatten
+  end
+
+  def generate_memo_lists
+    @memo_lists
+  end
+
+  def create_new_memo(title, content)
+    # latest_idキーからvalueを取得して、カウントをプラス１（初回投稿時には、ID1を作成）する
+    @latest_memo_id = @latest_memo_id.zero? ? 1 : @latest_memo_id + 1
```

Comment:
0 + 1 = 1なので条件分岐は不要そうです。

```suggestion
    @latest_memo_id += 1
```

Reply:
確かに初期値が「0」なので、条件分岐は不要ですね。
修正いたします。